src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -397,16 +398,46 @@ public String getQuestionResultStatisticsHtml(
         Map<String, List<Integer>> optionPoints = generateOptionPointsMapping(responses);
 
         DecimalFormat df = new DecimalFormat("#.##");
+        /*
+        Map<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>(new Comparator<String>() {
+            @Override
+            public int compare(String s1, String s2) {
+                return s2.compareTo(s1);
+            }
+        });
+        */
+        Map<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>();
+        
+        Map<String, String> identifierMap = new HashMap<String, String>();
         
         for (Entry<String, List<Integer>> entry : optionPoints.entrySet()) {
             
+            if (distributeToRecipients) {
+                
+                String participantIdentifier = entry.getKey();
+                String name = bundle.getNameForEmail(participantIdentifier);
+                
+                identifierMap.put(name, participantIdentifier);
+                sortedOptionPoints.put(name, entry.getValue());
```

Comment:
Hmm if there are 2 or more recipients with the same name, this would result in problems, although I'm not quite sure the best way to solve it myself. Perhaps one way could be to let the key be a concatenation of the recipient name and email. @kanghj any suggestions on how to overcome this?